exercises/concept/ellens-alien-game/.docs/hints.md

```diff
@@ -2,35 +2,40 @@
 
 ## 1. Create the Alien Class
 
-- Remember that object methods are always passed `self` as the first parameter.
-- Remember the double underscores on _both_ sides of `__init__`.
-- Instance variables are unique to the class instance (object) that possesses them.
-- Class variables are the same across all instances of a class.
+- Remember that `object methods` are always passed `self` as the first parameter.
+- Remember the double underscores on _both_ sides of `__init__()`.
+- Instance variables are unique to the `class` instance (_object_) that possesses them.
+- Class variables are the same across all instances of a `class`.
 
 ## 2. The `hit` Method
 
-- Remember that object methods are always passed `self` as the first parameter.
+- Remember that `object methods` are always passed `self` as the first parameter.
+- You can choose to allow the Alien's health to fall below zero, or require that it does not.
 
 ## 3. The `is_alive` Method
 
-- Remember that object methods are always passed `self` as the first parameter.
-- 0 is not the only 'dead' condition.
+- Remember that `object methods` are always passed `self` as the first parameter.
+- 0 may not be the only 'dead' condition, depending on how `hit()` is implemented.
 
 ## 4. The `teleport` Method
 
-- Remember that object methods are always passed `self` as the first parameter.
+- Remember that `object methods` are always passed `self` as the first parameter.
+- Instance attributes can be updated from a method by using `self.<attribute>` = `<new attribute value>`.
 
 ## 5. The `collision_detection` Method
 
-- Remember that object methods are always passed `self` as the first parameter.
-- This method seems like an excellent place to use some kind of placeholder.
+- Remember that `object methods` are always passed `self` as the first parameter.
+- This method seems like an excellent place to use some kind of placeholder...
```

Comment:
```suggestion
- This method seems like an excellent place to use some kind of placeholder…
```

Reply:
I think my ellipsis use is in-bounds, but ... ok. 😉

Reply:
It was definitely optional.  (hence approved but commented.)